src/tests/multihost/alltests/script/nfs_server_setup.sh

```diff
@@ -0,0 +1,9 @@
+mkdir -p /export/project{1..2}
```

Comment:
Please do not introduce random shell files.
All of this can be done via run_command in a helper function.